### DISCUSSION TITLE
What backend platform to use with Flutter_Data?

### DISCUSSION THREAD

**Discussion by arqeco** [flutterdata/flutter_data#256] [General]
Hello,

What backend platform may you suggest to using with Flutter_Data?

Platforms like Supabase and Appwrite looks very interesting since they have a lot of things ready to use like authentication and easy ways to create an database and API.

I am looking for something that demands more administering than programming in the backend, and that is Flutter_Data compatible.

But if there is some very good solution for the backend that requires programming and works well with Flutter_Data I would like to know it too.

Thank you

**Comment by bahady1** [comment]
You can use any Back end framework such as Laravel or Django. I had create an application using Flutter and Laravel which help to learn new stuffs.
